docs/translations/translations.json

```
@@ -22,7 +22,7 @@
     "refNotHeld": "The component cannot hold a ref.",
     "refRootElement": "The <code>ref</code> is forwarded to the root element.",
     "ruleName": "Rule name",
-    "spreadHint": "Any other props supplied will be provided to the root element ({{spreadHintElement}}).",
+    "spreadHint": "Props of the {{spreadHintElement}} are also available.",
```

Maybe to give a notion of priority? It's going to be first the props in the table, then, the spreaded props.

```suggestion
    "spreadHint": "In addition to the props detailed below, the props of the {{spreadHintElement}} are also available.",
```

---

Add "component" after `{{spreadHintElement}}`?

---

> Maybe to give a notion of priority? It's going to be first the props in the table, then, the spreaded props.

This is too wordy. It's pretty clear that the "other props" are the one in the same context i.e. directly below.


> Add "component" after {{spreadHintElement}}?

This would clash when we have a native element:

```diff
-Props of the native element are also available.
+Props of the native element component are also available.
```

-- https://deploy-preview-26778--material-ui.netlify.app/api/form-control/#main-content

I'd be happy with `native component` though i.e. `spreadHintElement` for host components will be `native` instead of `native element`.